source/code/plugins/in_npmd_server.rb

```diff
@@ -207,27 +209,32 @@ def npmd_reader
                     _json = check_and_get_json(_line.chomp)
                     unless !_json.nil?
                         Logger::logWarn "Sent string to plugin is not a json string", Logger::loop
+                        log_error "String received not json: #{_line[0..100]}" if _line.bytesize > 50
```

Comment:
we need to be wary of the 17 bytes case. Anything above that is OK. So went with 50.